lib/puppet/application/cert.rb

```diff
@@ -79,6 +79,22 @@ def subcommand=(name)
     set_log_level
   end
 
+  option("--human-readable", "-h") do |arg|
```

Comment:
Yes, though the help action will happen if there are no other subcommands listed. So:
```
[root@d7s7xpfrn9lqu1o ~]# puppet cert -h

puppet-cert(8) -- Manage certificates and requests
========

SYNOPSIS
--------
Standalone certificate authority. Capable of generating certificates,
but mostly used for signing certificate requests from puppet clients.

```

While if a subcommand is listed that will take precedence:
```
[root@d7s7xpfrn9lqu1o ~]# puppet cert list -h
  "pp14.delivery.puppetlabs.net"
  (SHA256) 7B:59:1D:6E:D9:20:76:9F:B0:6E:98:43:8A:54:30:D6:2B:E8:59:55:1C:BF:E7:BF:A1:38:18:9A:EF:55:92:20
    Status: Request Pending
    Extensions:
      challengePassword: "386587c8bed7657e4177c1b59af55ec0"
      pp_auth_role: "puppet infrastructure"
      pp_authorization: "true"
      pp_uuid: "be9d197b-ffee-4caa-9867-f7ce7285a23f"

....
```
Though `--help` still works:
```
[root@d7s7xpfrn9lqu1o ~]# puppet cert list --help

puppet-cert(8) -- Manage certificates and requests
========

SYNOPSIS
--------
Standalone certificate authority. Capable of generating certificates,
but mostly used for signing certificate requests from puppet clients.

....
```
It's a really good point, Josh, and I don't specifically remember us discussing it during UX testing. We did discuss that `-h` was a our preferred flag for human readable output since there is precedent in `ls`, `df`, etc.. I also believe everyone used `puppet help cert` when looking for help with the tool.

FWIW, `-h` does not work for puppet itself:
```
[root@d7s7xpfrn9lqu1o ~]# puppet -h
Error: Could not parse application options: invalid option: -h
```

I have a slight preference for overriding the short help flag and removing the reference to `-h` being used for help in the usage summary rather than changing the short flag for human readable content. But, I don't have a strong preference and would love to hear folks' feedback.